src/test/com/abstract_calss/Human.java

```diff
@@ -0,0 +1,12 @@
+package test.com.abstract_calss;
+
+public class Human extends Animal{
+    @Override
+    void move() {
+        System.out.println("walk");
+    }
+    @Override
+    void talk() {
+
```

Comment:
I suppose you're assuming that you have a technical task end everything that not there is the fault of the person who prepared it. Development is a teamwork, if you want to pretend that everybody around you are idiots, nobody will work with you. So it's your job to ask questions, and to help other people to do their job better and it's their work to help you doing your work better, that's a teamwork, everything else is considered as toxicity and people are being faired because of it much often that because their technical skills aren't good enough.